addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -1317,6 +1321,7 @@ export class PosStore extends Reactive {
     }
     showScreen(name, props) {
         this.previousScreen = this.mainScreen.component?.name;
+        name = !name || name == "" ? this.firstScreen : name;
```

Comment:
if `name == ""` then `!name` will give `true` and thus the second part of the `||` (`name == "" ? this.firstScreen : name`) will never be computed
```suggestion
        name = !name ? this.firstScreen : name;
```